ahl_health_apps_analysis/pipeline/README.md

```diff
@@ -1,29 +1,42 @@
-	- app_details_ratings saved in location **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_ratings.csv'**
-	- app_details_price (dataset that only includes the apps that apply installation charges) saved in location **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_price.csv')**
-	- app_details_installations_joined (dataset includes information on precentage change from number of installations of apps released in 2019 to number of installations of apps released in 2021 averaged by cluster. Dates chosen to see the effect covid might of had on behaviour of customers and developers.) saved in location **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_installations.csv'**
-	- app_details_release_joined (dataset includes information on percentage change from number of apps released 2019 to number of apps released 2019 avereged by cluster. Dates chosen to see the effect covid might of had on behaviour of customers and developers) **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_release_joined.csv'**
-
+   - app_details_ratings saved in location **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_ratings.csv'**
+   - app_details_price (dataset that only includes the apps that apply installation charges) saved in location **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_price.csv')**
+   - app_details_installations_joined (dataset includes information on precentage change from number of installations of apps released in 2019 to number of installations of apps released in 2021 averaged by cluster. Dates chosen to see the effect covid might of had on behaviour of customers and developers.) saved in location **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_installations.csv'**
+   - app_details_release_joined (dataset includes information on percentage change from number of apps released 2019 to number of apps released 2019 avereged by cluster. Dates chosen to see the effect covid might of had on behaviour of customers and developers) **f'outputs/data/{date}-{cluster_type}-{clusters}-app_details_release_joined.csv'**
 
 ## Adjust parameter in each file before running each file to suit your specific aims.
+
 Default Parameters:
 **date** = ('2022-11-28')
 **clusters** = 20
 **cluster_type** = kmeans
 
-
 Default name for final manually refined dataset is saved in the location **f'outputs/data/{date}-{cluster_type}-{clusters}-refined.csv'** completed in step 5 of the pipeline.
-The final dataset before manually adjusting clusters and names is the dataset saved in the location **f'outputs/data/{date}-{cluster_type}-{clusters}-cluster.csv'** completed in step 4 of the pipeline.
\ No newline at end of file
+The final dataset before manually adjusting clusters and names is the dataset saved in the location **f'outputs/data/{date}-{cluster_type}-{clusters}-cluster.csv'** completed in step 4 of the pipeline.
```

Comment:
sorry @imy99 I didnt mean to change all these formatting bits to your readme - the cookie cutter automatically did it!